.buildbot.sh

```diff
@@ -2,27 +2,15 @@
 
 set -e
 
+YKRUSTC_REPO=https://github.com/softdevteam/ykrustc
```

Comment:
I must admit, this makes the script harder to read IMHO: it's only used in one place, and the code that comes after it heavily depends on the repo being called `ykrustc`. I'd suggest inlining this variable TBH.